matrix/plugins/actions.py

```diff
@@ -0,0 +1,45 @@
+from datetime import timedelta, datetime
+
+from .glitch import glitch  # noqa
```

Comment:
This import was so that you could have:

```yaml
action: matrix.plugins.actions.deploy
action: matrix.plugins.actions.health
action: matrix.plugins.actions.glitch
```

I also considered making the `matrix.plugins.actions` namespace the default.  Mostly, this was because I felt like `matrix.plugins.glitch.glitch` was a bit unclear, but I'm not married to this structure if we want to avoid the import.